frappe/utils/backups.py

```diff
@@ -593,10 +570,11 @@ def new_backup(
 	return odb
 
 
-def delete_temp_backups(older_than=24):
+def delete_temp_backups():
```

Comment:
Can we still accept `older_than` here? Would still keep this function re-usable.

Everything else, LGTM

Reply:
Sure, but the config still gets priority, right?